cnd/src/btsieve/ethereum.rs

```diff
@@ -15,7 +15,7 @@ pub enum EthereumQuery {
     Transaction {
         from_address: Option<Address>,
         to_address: Option<Address>,
-        is_contract_creation: Option<bool>,
```

Comment:
This change is semantically incorrect :)
We need three states: None, true and false.

With the proposed change, we cannot express a query where we "don't care" about whether it is a contract creation or not.